code/mob/living/carbon/human.dm

```diff
@@ -465,6 +465,25 @@
 			return
 		return 0
 
+	proc/randomize(var/target, var/mob/user, var/show_message = 1)
+		var/number_replaced 
```

Comment:
`.` please

Reply:
I did actually originally implemented this by just using `.`, but I got some weird compilation errors that I wasn't able to resolve.

I probably messed up something without realizing it, so I'll give it another shot.

Reply:
Yeah, guess I fucked something up, because I fixed it now.